### DISCUSSION TITLE
Reconnecting...

### DISCUSSION THREAD

**Discussion by jmccarthy6** [sorayuki/obs-multi-rtmp#266] [General]
I get this when I try to stream to kick.com, I have the stream url and stream key set but it just sits here reconnecting. Then for a split second it will show like FPS 0 and time start counting but just immediately goes to reconnecting. Any help is appreciated. 

![image](https://user-images.githubusercontent.com/105233588/228975456-d38259f8-3a77-47d8-bf61-d83fc746aea3.png)

![image](https://user-images.githubusercontent.com/105233588/228975843-be813ea7-df2b-4351-860c-433adf2c2153.png)

**Comment by gaditanorl** [comment]
Same for me, it happens when i want to stream to kick, but it happens also from the main obs "start streamming". Maybe the problem is the platform, or the obs.
